std/typecons.d

```diff
@@ -792,10 +852,36 @@ template Tuple(Specs...)
          * Returns:
          *     The string representation of this `Tuple`.
          */
-        string toString()()
+        string toString()() const
         {
-            import std.conv : to;
-            return this.to!string;
+            import std.array : appender;
+            auto app = appender!string();
+            this.toString((const(char)[] chunk) => app ~= chunk);
+            return app.data;
+        }
+
+        ///
+        unittest
+        {
+            import std.format    : format, FormatException;
+            import std.format : format;
```

Comment:
It is redundant. I've deleted it, but it's not worth pushing and letting the auto-tester do it's work all again.
The real question is: Do we really want to have `assertThrown` in the unit test?